rumqttc/src/lib.rs

```diff
@@ -121,12 +121,21 @@ pub type Incoming = Packet;
 /// Current outgoing activity on the eventloop
 #[derive(Debug, Eq, PartialEq, Clone)]
 pub enum Outgoing {
-    /// Publish packet with packet identifier. 0 implies QoS 0
-    Publish(u16),
-    /// Subscribe packet with packet identifier
-    Subscribe(u16),
-    /// Unsubscribe packet with packet identifier
-    Unsubscribe(u16),
+    /// Publish packet with packet identifier and topics. 0 implies QoS 0
+    Publish {
+        pkid: u16,
+        topics: Vec<String>,
```

Comment:
Why does publish have a list of topics instead of 1 topic?

Reply:
I didn't realise publish can only have one. I've made it a single topic now.